### ISSUE TITLE
Cannot navigate back if activity is opening from beagle 

### ISSUE THREAD

**Issue by egorikftp** [pandulapeter/beagle#16]
Hi)
I faced an interesting issue. In my app, I have Trick Button and by clicking some debug Activity is opening.

```kotlin
Trick.Button(
                text = "test",
                onButtonPressed = {
                    Beagle.currentActivity?.run {
                        startActivity(Intent(this, MotionActivity::class.java) )
                    }
                }
            )
```

After that, I cannot go back from MotionActivity 🙂 

Looks like a problem in this block of code

```Kotlin
private val onBackPressedCallback = object : OnBackPressedCallback(false) {
        override fun handleOnBackPressed() {
            dismiss()
        }
    }
```

Thank you.

**Comment by pandulapeter** [comment]
Thanks for the bugreport!

I've managed to reproduce the issue. I'm only using Beagle in single-Activity projects, that's why it was not detected before, sorry about it.

The core problem is that the drawer should be visible when you open MotionActivity (as it was never dismissed). The onBackPressedCallback you mentioned gets enabled or disabled in function of the drawer's state and apparently when new Activity is started while the drawer is open, things get out of sync.

I'll try to find a solution before the next release, but for the time being, manually calling 'Beagle.dismiss()' right after the 'startActivity()' call should fix the issue.

**Comment by egorikftp** [comment]
Hello.
Proposed workaround works as expected)

Once again, thanks for the useful library 👍

**Comment by pandulapeter** [comment]
Should be fixed in version 1.7.1 - now Beagle will automatically get dismissed if a new Activity is opened on top of the current one.
